README.md

```diff
@@ -1 +1 @@
-docs/index.md
\ No newline at end of file
+./docs/index.md
```

Comment:
This is not needed but I can overlook if you don't have time to fix it.

Reply:
Problem is in symlink?

Reply:
No, the addition of `./`. I am just nitpicking here.